sdk/test/Services/DynamoDBv2/UnitTests/Custom/DocumentModel/TableTests.cs

```diff
@@ -0,0 +1,266 @@
+﻿using System.Collections.Generic;
+using Amazon;
+using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.DocumentModel;
+using Amazon.DynamoDBv2.Model;
+using Amazon.Runtime;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace AWSSDK_DotNet.UnitTests
+{
+    [TestClass]
+    public class TableTests
+    {
+
+        private Mock<IAmazonDynamoDB> _ddbClientMock;
+        private Table _table;
+        private string _tableName = "TestTable";
+
+        [TestInitialize]
+        public void Setup()
+        {
+            _ddbClientMock = new Mock<IAmazonDynamoDB>(MockBehavior.Strict);
+            // Mock IClientConfig
+            var clientConfigMock = new Mock<IClientConfig>();
+            // Setup any properties/methods you expect to be used, e.g.:
+            clientConfigMock.SetupGet(c => c.RegionEndpoint).Returns((RegionEndpoint)null);
+            clientConfigMock.SetupGet(c => c.ServiceURL).Returns((string)null);
+            // Add more setups as needed for your tests
+
+            // Setup the Config property on the IAmazonDynamoDB mock
+            _ddbClientMock.SetupGet(c => c.Config).Returns(clientConfigMock.Object);
+
+            var config = new TableConfig(_tableName);
+
+            _table = new Table(_ddbClientMock.Object, config);
+            _table.ClearTableData();
+            _table.Keys.Add("Id", new KeyDescription { IsHash = true, Type = DynamoDBEntryType.String });
+            _table.HashKeys.Add("Id");
+        }
+
+        [TestMethod]
+        public void UpdateHelper_StandardUpdate_SendsCorrectRequest()
+        {
+            // Arrange
+            var doc = new Document { ["Id"] = "1", ["Name"] = "Test" };
+            var key = new Key { ["Id"] = new AttributeValue { S = "1" } };
+            var config = new UpdateItemOperationConfig();
+
+            _ddbClientMock
+                .Setup(c => c.UpdateItem(It.Is<UpdateItemRequest>(r =>
+                    r.TableName == _tableName &&
+                    r.Key.ContainsKey("Id") &&
+                    r.AttributeUpdates != null &&
+                    r.AttributeUpdates.ContainsKey("Name") &&
+                    r.UpdateExpression == null
+                )
+                ))
+                .Returns(new UpdateItemResponse { Attributes = new Dictionary<string, AttributeValue>() });
+
+            // Act
+            var result = _table.UpdateHelper(doc, key, config, null);
+
+            // Assert
+            Assert.IsNull(result);
```

Comment:
nit: can you add a comment/update the test name to be more clear?  "sendscorrectrequest" -> "sendsSomethingSomething". im not sure what "correct" means here